lib/rubrowser/server.rb

```diff
@@ -2,6 +2,7 @@
 require 'erb'
 require 'rubrowser/data'
 require 'rubrowser/formatter/json'
+require 'byebug'
```

Comment:
byebug is a development dependency so then we include it will raise error when someone is installing the gem normally from rubygems as the development dependencies are not installed in this case.